hopr/hopr-lib/src/lib.rs

```diff
@@ -830,6 +831,28 @@ impl Hopr {
             }
         }
 
+        // Check Safe-module configuration:
+        // 1) if the node is already included into the module
+        // 2) if the module is enabled in the safe
+        // 3) if the safe is the owner of the module
+        // if any of the conditions is not met, node panic
+        let safe_module_configuration = self
+            .chain_api
+            .rpc()
+            .check_node_safe_module_configuration(self.me_onchain())
+            .await
+            .map_err(HoprChainError::Rpc)?;
+
+        if !safe_module_configuration.should_pass() {
+            error!(
+                "Something is wrong with the safe module configuration: {:?}",
+                safe_module_configuration
+            );
```

Comment:
If I saw this error, I would be unhappy. Also, don't put the log here pls, that's a library call returning error. Let's put the error at the caller's position.